tests/e2e/utils/withFailTimeout.ts

```diff
@@ -1,12 +1,12 @@
 import CONFIG from '../config';
 
-const TIMEOUT = process.env.INTERACTION_TIMEOUT || CONFIG.INTERACTION_TIMEOUT;
+const TIMEOUT = Number(process.env.INTERACTION_TIMEOUT ?? CONFIG.INTERACTION_TIMEOUT);
```

Comment:
```suggestion
const TIMEOUT = Number(process.env.INTERACTION_TIMEOUT || CONFIG.INTERACTION_TIMEOUT);
```

I think we should keep `||` if `process.env.INTERACTION_TIMEOUT` can be empty string.

With `||`

process.env.INTERACTION_TIMEOUT -> `''`
CONFIG.INTERACTION_TIMEOUT -> `'1000'`

```ts
'' || '1000' // result is '1000' ✅ 
```

With `??`

```ts
'' ?? '1000' // result is '' ❌ 
```

Reply:
That would require putting
```// eslint-disable @typescript-eslint/prefer-nullish-coalescing```

above the line, but we probably should do that anyway

Reply:
Done! [9561f75](https://github.com/Expensify/App/pull/38923/commits/9561f7509874b62b5de42b45e95b6a5107adfe6d)